client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/logging/memory/MemLogStorage.java

```diff
@@ -101,15 +97,12 @@ public void addLogRecord(LogRecord record) {
             consumedVolume += record.getSize();
         }
         LOG.trace("Added a new log record to bucket [{}]", currentBucket.getId());
+        return new BucketInfo(currentBucket.getId(), currentBucket.getCount());
     }
 
     @Override
-    public LogBlock getRecordBlock(long blockSize, int batchCount) {
-        LOG.trace("Getting new record block with block size = {} and count = {}", blockSize, batchCount);
-        if (blockSize > maxBucketSize || batchCount > maxBucketRecordCount) {
-            //TODO: add support of block resize
-            LOG.warn("Resize of record block is not supported yet");
-        }
+    public LogBlock getRecordBlock() {
```

Comment:
I think this method should be renamed/changed to 
public Bucket getNextBucket();
This will help to avoid confusuions betweeb BucketInfo and LogBlock. LogBlock is actually a Bucket. correct?